ios/Classes/Util/Base64Coder.m

```diff
@@ -0,0 +1,10 @@
+#import "Base64Coder.h"
+
+@implementation Base64Coder
+
++ (NSString *)base64StringFromData:(NSData *)data {
```

Comment:
These utilities are only used in BLEmulator, they should be BLEmulator directory

Reply:
Good point

Reply:
Moved to the Blemulator part of the project

Reply:
There are two more classes I think that need to be moved.

Reply:
Moved